src/ParkInfo/ParkInfo.tsx

```diff
@@ -2,16 +2,22 @@ import './ParkInfo.scss'
 import About from '../About/About'
 import Location from '../Location/Location'
 import Contact from '../Contact/Contact'
+import BannerIcons from '../BannerIcons/BannerIcons'
 import { Switch, Route } from 'react-router-dom'
-import { CurrentParkContainer } from '../interfaces'
-import { RouteComponentProps } from 'react-router-dom'
+import { AddRemoveFunctionality, CurrentParkContainer } from '../interfaces'
 
-type ParkInfoProps = CurrentParkContainer | RouteComponentProps
+type ParkInfoProps = CurrentParkContainer | AddRemoveFunctionality | { localPark: () => any | void}
```

Comment:
This is cool. I didn't realize that we can assign a type in here without it being an interface